README.md

```diff
@@ -39,6 +39,10 @@ Include the CSS and JavaScript
 
 ## Installation
 
+To install GOV.UK Frontend NPM package in your application, need to create a [package.json](https://docs.npmjs.com/files/package.json) file in your application (most often in the root of your application).
```

Comment:
'**you** need to create'

I'm not sure if this is the best way to talk about it. I *think* `npm install` would do this for you, or maybe `npm init` and then `npm install`

Reply:
they need to create it, but yes can can add a line than it can be created with `npm init`